packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,505 @@ export const SettingsOverviewExample = (): JSX.Element => {
+                Set up destination
+              </Heading>
+            </Box>
+            <Box>
+              <Form>
+                <FormControl>
+                  <Label htmlFor={input1}>Destination name</Label>
+                  <Input type="text" id={input1} name="input1" />
+                  <HelpText>
+                    Identifies this destination within your organization and typically includes the business unit,
+                    geographic region, or environment. Example: Google Analytics North America Prod.
+                  </HelpText>
+                </FormControl>
+              </Form>
+            </Box>
+            <Box display="flex" justifyContent="space-between">
+              <Button variant="secondary">
+                <ArrowBackIcon decorative />
+                Back
+              </Button>
+              <Box display="flex" columnGap="space40">
+                <Button variant="secondary">Cancel or Close</Button>
+                <Button variant="primary">Create destination</Button>
+              </Box>
+            </Box>
+          </Box>
+        </Box>
+      </SidebarPushContentWrapper>
+    </>
+  );
+};
+DefaultWizardWithContentExample.parameters = {
+  padding: false,
+};
+
+export const DefaultWizardOrderedListExample = (): JSX.Element => {
+  const input1 = useUID();
+  const input2 = useUID();
+  const input3 = useUID();
+  const input4 = useUID();
+  const input5 = useUID();
+  return (
+    <Box display="flex" width="100%" justifyContent="center" paddingX="space10">
+      <Box
+        paddingTop="space130"
+        paddingBottom="space160"
+        display="flex"
+        flexDirection="column"
+        rowGap="space130"
+        width="size70"
+        maxWidth="size80"
+      >
+        <Box display="flex" flexDirection="column">
+          <ProgressSteps>
+            <ProgressStepCurrent as="div">Label</ProgressStepCurrent>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+          </ProgressSteps>
+          <Box marginTop="space70">
+            <DetailText>Add warehouse</DetailText>
+            <Heading as="h4" variant="heading10">
+              Connect Azure SQL data warehouse
+            </Heading>
+            <Paragraph marginBottom="space0">
+              Follow the steps below to start setting up Azure SQL data warehouse.
+            </Paragraph>
+          </Box>
+        </Box>
+        <Box display="flex" flexDirection="column" rowGap="space130">
+          <Form>
+            <Box display="flex">
```

Comment:
question: Do you know why this happens? Size is set to `size10`, which should be 88px, but shrinks down to ~78px.

![Screenshot 2023-11-14 at 2 00 07 PM](https://github.com/twilio-labs/paste/assets/3821840/8ad176f7-0e23-4c5c-b7eb-e64fbe5e655f)

Reply:
fixed by adding flexShrink: 0 to the box with the number